Formula/gst-plugins-bad.rb

```diff
@@ -48,13 +48,13 @@ def install
       --disable-debug
       --disable-dependency-tracking
     ]
-
+    # UPDATE Apple Media support works in 1.10.2 for Mavericks.
     # upstream does not support Apple video for older SDKs
     # error: use of undeclared identifier 'AVQueuedSampleBufferRenderingStatusFailed'
     # https://github.com/Homebrew/legacy-homebrew/pull/35284
-    if MacOS.version <= :mavericks
-      args << "--disable-apple_media"
-    end
+    # if MacOS.version <= :mavericks
+    #   args << "--disable-apple_media"
+    # end
```

Comment:
Rather than commenting this out please remove all these lines (including your new comment). Thanks!